tools/scrumcheck/ui/assets/app.js

```
@@ -0,0 +1,1066 @@
+        const raw = String(value == null ? '' : value).trim();
+        if (!raw) return '';
+        try {
+          const parsed = new URL(raw, window.location.origin);
+          if (parsed.protocol === 'http:' || parsed.protocol === 'https:') {
+            return parsed.href;
+          }
+        } catch (_) {}
+        return '';
+      }
+
+      function renderUnreleasedItem(item, cssClass) {
```

ESLint

[error] Delete `····`

(prettier/prettier)